arbeitszeit_flask/translations/de/LC_MESSAGES/messages.po

```diff
@@ -852,6 +863,18 @@ msgstr "Noch keine Arbeiter*innen registriert"
 msgid "Registration of productive consumption"
 msgstr "Registrierung produktiven Konsums"
 
+#: arbeitszeit_flask/templates/company/register_productive_consumption.html:46
+msgid "Plan name"
+msgstr "Planname"
```

Comment:
those put-together nouns look strange in german imo. Can we not get rid of them at all, and simply use "Name-Name" and "description-Beschreibung"? This maybe could be re-used at other places as well. But of course, it would need to be clear from the context that, e.g. name of a PLAN is referred to.

Reply:
see comment below